woocommerce/woocommerce-ios
# Home Screen quick actions

## Description


Added [home screen quick actions](https://developer.apple.com/documentation/uikit/menus_and_shortcuts/add_home_screen_quick_actions):
- New Order
- Orders
- Add a product
- Collect Payment

## Testing instructions

- press and hold the Woo icon on the home screen
- you should get 4 actions, as described above
- Test all by tapping them and make sure that each one does what it says

## Screenshots


![Simulator Screenshot - iPhone 15 Pro - 2024-03-20 at 10 10 22](https://github.com/woocommerce/woocommerce-ios/assets/6242034/e1848683-0b98-4b7e-8f3f-1c3b6ad16682)

| New Order      | Orders  |
| ----------- | ----------- |
| ![Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 04](https://github.com/woocommerce/woocommerce-ios/assets/6242034/0e0d0237-cf42-4571-8fc2-18b85fcae8ae)      | ![Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 16](https://github.com/woocommerce/woocommerce-ios/assets/6242034/27d4dc0e-5cb3-4a46-baa5-f5530dcc3440)       |

| Add a product      | Collect Payment  |
| ----------- | ----------- |
| ![Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 27](https://github.com/woocommerce/woocommerce-ios/assets/6242034/fc4f592b-aafe-430d-8df2-0762d1c777f5) | ![Simulator Screen Recording - iPhone 15 Pro - 2024-03-20 at 10 13 38](https://github.com/woocommerce/woocommerce-ios/assets/6242034/0f5996e8-e305-4ea9-afa4-5c8ffaac59b3) |



---
